cloudconfig/cloudinit/cloudinit_centos.go

```diff
@@ -254,3 +256,9 @@ func (cfg *centOSCloudConfig) addRequiredPackages() {
 //addPackageProxyCmds. Research if calling the same is fine.
 func (cfg *centOSCloudConfig) updateProxySettings(proxySettings proxy.Settings) {
 }
+
+// AddNetworkConfig is defined on the NetworkingConfig interface.
+// TODO(wpk) This has to be implemented for CentOS on VSphere to work properly!
```

Comment:
Is this really something we can do / control?

Reply:
Yes, we can generate /etc/network/ifcfg-ethX for Centos - but IMHO until someone asks as to do so (or someone is bored has too much time to spare) it's not worth doing.